datasm/resources/dataset_spec.yaml

```diff
@@ -352,6 +424,52 @@ project:
                         - gpp
                         - rh
                         - ra
+            E3SM-2-0:
+                hist-aer:
+                    start: 1850
+                    end: 2020
+                    ens:
+                        - r1i1p1f1
+                        - r2i1p1f1
+                        - r3i1p1f1
+                        - r4i1p1f1
+                        - r5i1p1f1
+                    except:
+                        - cLitter
+                        - cProduct
+                        - cSoilFast
+                        - cSoilMedium
+                        - cSoilSlow
+                        - fFire
+                        - fHarvest
+                        - cVeg
+                        - nbp
+                        - gpp
+                        - rh
+                        - ra
+                hist-GHG:
+                    start: 1850
+                    end: 2020
```

Comment:
I think end should be 2014 here.

Reply:
Ok, easy to change.

Question is, if the "CMIP required" says one thing, but different teams produce different numbers of years, what should go into the dataset spec?  I agreed with "full years supplied", but that might not be a constant.  On the one hand, I'd like datasm to "automatically" trim excess files from warehouse extraction (they would still remain in archive), but now I'm not sure (in general).   I'll edit these, in any case.